wren-ui/src/components/pages/home/thread/StepContent.tsx

```diff
@@ -108,6 +108,7 @@ export default function StepContent(props: Props) {
             CollapseContentProps,
             'attributes'
           >)}
+          stepIndex={stepIndex}
```

Comment:
The**collapseContentProps** doesn't have the **stepIndex** props, so we must omit it.

```diff
         <CollapseContent
           {...(collapseContentProps as Omit<
             CollapseContentProps,
-            'attributes'
+            'attributes' | 'stepIndex'
           >)}
           stepIndex={stepIndex}
           key={`collapse-${stepNumber}`}
```